packages/gatsby-plugin-functions/src/gatsby-node.ts

```diff
@@ -129,26 +126,43 @@ export async function onCreateDevServer(
   )
   const files = await glob(functionsGlob, { cwd: functionsDirectory })
 
-  // console.log({ files })
+  reporter.info(`Attaching Functions to development server.`)
 
   const knownFunctions = new Map(
     files.map(file => [
       urlResolve(path.parse(file).dir, path.parse(file).name),
       file,
     ])
   )
-  // console.log(app, functionsDirectoryPath)
 
   app.use(`/api/:functionName`, (req, res, next) => {
     const { functionName } = req.params
 
     if (knownFunctions.has(functionName)) {
-      const fn = require(path.join(
-        functionsDirectory,
-        knownFunctions.get(functionName) as string
-      ))
+      const activity = reporter.activityTimer(
+        `Executing function ${functionName}`
+      )
+      activity.start()
+      const compiledFunctionsDir = path.join(
+        process.cwd(),
+        `.cache`,
+        `functions`
+      )
+      const funcNameToJs = (knownFunctions.get(functionName) as string).replace(
+        `ts`,
+        `js`
+      )
```

Comment:
We should get this information from webpack instead. It's fine for now :)

Reply:
how?